### PR TITLE
Cards

### LINKED ISSUE #31: Integrar interfaz de gestión de tarjetas en cuenca-python

### PR DESCRIPTION
**Pull request by felipaoo**
closes #31

### CODE REVIEW

File: cuenca/resources/base.py [hunk 1]
```diff
@@ -71,8 +81,8 @@ class Queryable(Resource):
 
     @classmethod
     def one(cls, **query_params) -> Resource:
-        q = cls._query_params(limit=2, **query_params)
-        resp = session.get(cls._resource, q.dict())
+        query = cls._query_params(limit=2, **query_params)
+        resp = session.get(cls._resource, query.dict(exclude_none=True))
```
Comment (matin) [inline cuenca/resources/base.py:85]: @felipaoo Why did you change this, and why is it inconsistent with `.all()`, `.first()` and `.count()`?

Comment (matin) [reply cuenca/resources/base.py:85]: If the idea is to exclude `None` to all queries, you should change the bas3 `QueryParams` in cuenca-validations to something like this:

```python
from pydantic import BaseModel

class QueryParams(BaseModel):
    def dict(self, exclude_none=True, **kwargs):
        return super().dict(exclude_none=exclude_none, **kwargs)
```

Comment (matin) [reply cuenca/resources/base.py:85]: in fact ... it's already there! https://github.com/cuenca-mx/cuenca-validations/blob/main/cuenca_validations/types/queries.py#L28-L33

Please remove your change. It's entirely redundant.

Comment (felipaoo) [reply cuenca/resources/base.py:85]: removed
